core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -103,6 +103,8 @@ object ConsoleConsumer extends Logging {
         consumer.stop()
 
         shutdownLatch.await()
+
+        System.out.println("shutdown_complete")
```

Comment:
Yeah, that might work, but it seems a little weird for a test's correctness to depend on the log level. Do we do that elsewhere?